### PR TITLE
Add support for indexing audio:create transactions

### LINKED ISSUE #20: Add support for indexing Audio create command
### Description

As a part of #2, we should be able to index the audio:create transactions and list the Audio NFTs. 

### Acceptance Criteria

- Index transactions of audio:create 
- List audio NFTs
- Filter audios by
   - audioID
   - creatorAddress
   - offset
   - count
- The above functionality must be tested (automated).

### PR DESCRIPTION
**Pull request by reyraa**
### What was the problem?
This PR resolves #20

### How was it solved?
- [x] Add basic support
- [x] Include owners
- [x] Include fit

*Note:* We decided to include genres in another ticket.

### How was it tested?
- [x] Update unit tests
- [x] Test NFT listing
- [x] Test transaction listing

### CODE REVIEW

File: services/blockchain-indexer/shared/database/schema/audios.js
Comment (haghighatcs) [inline services/blockchain-indexer/shared/database/schema/audios.js:1]: creatorAddress must not be nullable! so the audio should have at least one artist.
Don't you think so? @reyraa

File: services/blockchain-indexer/shared/indexer/transactionProcessor/audio/create.js
Comment (haghighatcs) [inline services/blockchain-indexer/shared/indexer/transactionProcessor/audio/create.js:1]: I think is this block you have to add a logic to do for updating the shares: @reyraa 
const ownerInfo = {
				...owner,
				audioID: eventData.audioID,
				nonce: tx.nonce,
				shares: 0,
			};

File: services/gateway/apis/http-version3/swagger/definitions/audios.json
Comment (haghighatcs) [inline services/gateway/apis/http-version3/swagger/definitions/audios.json:1]: Every change looks great but fit must be spelled as "feat" @reyraa

**Review by haghighatcs — APPROVED** [review]
Thank you @reyraa  everything is looking great to us!

File: services/blockchain-indexer/shared/indexer/transactionProcessor/audio/create.js
Comment (reyraa) [reply services/blockchain-indexer/shared/indexer/transactionProcessor/audio/create.js:1]: setAttributes can not change the shares. Shares are manipulated using the transfer command.

**Pull request merged**